docs/UserGuide.md

```diff
@@ -86,12 +85,14 @@ to apply some refactoring.
 
 LSP4IJ provides **experimental**  support for [LSP Semantic Tokens](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_semanticTokens).
 
-### Enable semantic tokens
+### Language Server coloration
+
+LSP4IJ provides default color for semantic tokens that you can configure with `Editor / Color Scheme / Language Server`:

```

Comment:
LSP4IJ provides default colors for semantic tokens, configurable in

Reply:
fixed